project/Settings.scala

```diff
@@ -33,7 +33,7 @@ object Settings {
     "-language:existentials", // Existential types (besides wildcard types) can be written and inferred
     "-unchecked", // Enable additional warnings where generated code depends on assumptions.
     "-Xcheckinit", // Wrap field accessors to throw an exception on uninitialized access.
-    "-Xfatal-warnings", // Fail the compilation if there are any warnings.
+//    "-Xfatal-warnings", // Fail the compilation if there are any warnings.
```

Comment:
did you mean to remove this?

Reply:
ah good catch..this was for running scalafix..will revert back